src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -276,6 +283,10 @@
     }
 }
 
+.unified-share-form-container {
```

Comment:
Can we use a SUIT CSS format for this name, maybe `.bdl-UnifiedShareForm-container`?